src/app/home/events/events.component.html

```diff
@@ -11,8 +11,26 @@ <h5 class="title-event">{{ event.name }}</h5>
             </div>
             <div class="card-content">
               <img class="logo-sw" [src]="event.bannerUrl" [alt]="event.name">
-              <p class="place">{{ event.place }}</p>
-              <p class="date">{{ event.date.toDate() | date: 'mediumDate':'es' }}</p>
+              <div class="data">
+                <p>
+                  <i class="fas fa-map-marker-alt"></i>
+                  {{ event.place }}
+                </p>
+                <p>
+                  <i class="far fa-calendar-alt"></i>
+                  {{ event.date.toDate() | date: 'mediumDate':'es' }}
+                </p>
+                <p>
+                  <i class="fas fa-clock"></i>
+                  {{ event.date.toDate() | date: 'shortTime':'es' }}
+                </p>
+                <p *ngIf="event.price">
+                  <i class="fas fa-money-bill-alt"></i>
+                  <span class="price">
+                    {{ event.price }} {{ currency }}
+                  </span>
+                </p>
+              </div>
```

Comment:
You are not using place and date classes. These are meant to avoid having multiple lines to the event data that can cause the one card to be bigger than others. Please, review them and try to apply a similar logic to what you did.